go/client/chat_ui.go

```diff
@@ -178,7 +177,18 @@ func (c *ChatUI) getUnverifiedConvo(ctx context.Context, conv chat1.Conversation
 							Body: "<pending>",
 						},
 					},
-					ClientHeader:   txtMsg.ClientHeader,
+					// This is a misuse of MessageClientHeaderVerified,
+					// the contents have not been verified at all.
+					// It is only used to show the initial pending list in
+					// `keybase chat ls --async`
+					ClientHeader: chat1.MessageClientHeaderVerified{
+						Conv:         txtMsg.ClientHeader.Conv,
+						TlfName:      txtMsg.ClientHeader.TlfName,
+						TlfPublic:    txtMsg.ClientHeader.TlfPublic,
+						MessageType:  txtMsg.ClientHeader.MessageType,
```

Comment:
Should this be hardcoded to the text type? It looks like we're assuming that above.